gallery/workbook/Azure Monitor.json

```diff
@@ -257,13 +257,6 @@
 					"description": "Performance (query, export, ingestion, streaming) of all your Azure Data Explorer Clusters",
 					"name": "Azure Data Explorer Overview"
 				},
-				{
-					"id": "Workbooks/Activity Log/AtScale",
```

Comment:
Also can we reuse the same Activity Log directory instead of making a new ActivityLogInsights directory?

Reply:
I am not the owner of the old one, have no Idea what it was used for

Reply:
Let's reuse the current directory. I created the old activity logs insights as a placeholder.

Reply:
so, can you move my code? or you need me to do it?

Reply:
done